src/core/CCOpt.ml

```diff
@@ -54,10 +58,16 @@ let (<*>) f x = match f, x with
 
 let (<$>) = map
 
-let (<+>) a b = match a with
+let or_ b a = match a with
```

Comment:
I think it's a good idea to have this non-infix alternative, indeed. What worries me a bit is the weird order of the parameters (which I understand stems from the choice in `get_lazy`…)